etc/zinit/networkd.yaml

```diff
@@ -1,3 +1,4 @@
 exec: networkd -broker unix:///var/run/redis.sock -root /var/cache/modules/networkd
+test: ip link show zos
```

Comment:
Internet is the service responsible to create zos,not networkd. So this test will always pass right away and doesn't really bring anything.

Reply:
Yeah, actually i totally can take this out, i needed it before the refactoring. This just grantee that the zos bridge is created for monitoring, but now network monitoring is done by networkd itself. so I will remove that.